site/docs/concepts/architecture.md

```diff
@@ -17,10 +17,9 @@ Modules can provide read/write access or produce implicit copies that serve as l
 
-Once a developer submits an `FybrikApplication` CRD to Kubernetes the ApplicationController will make sure that all the specs are fulfilled and will make sure that the data is made accessible to the user according to the previously defined policies. The `FybrikApplication` holds metadata about the application such the data assets required by the application, the processing purpose and the method of access the user wishes (protocol e.g. S3 or Arrow flight). 
-It uses this information to check with external systems (4) if access or copy is allowed
-and whether restrictive policies such as masking or hashing have to be applied. It compiles blueprints based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.
-As data assets may reside in different systems the blueprints are compiled in a `Plotter` CRD (6) that specifies which blueprints have to be executed in which cluster.
+Once a developer submits an `FybrikApplication` custom resource to Kubernetes the ApplicationController will make sure that all the specs are fulfilled and will make sure that the data is made accessible to the user according to the previously defined policies. The `FybrikApplication` holds metadata about the application such the data assets required by the application, the processing purpose and the method of access the user wishes (protocol e.g. S3 or Arrow flight). 
+It uses this information to check with external systems (4) if access and/or copy is allowed
+and whether restrictive policies such as masking or hashing have to be applied. It generates a `Plotter` custom resource (6) based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.  The plotter contains the modules required and all the information required to deploy them on the cluster where they will run, as well as the flow of data between the asset and the workload through the chosen modules.  The plotter generates a `Blueprint` custom resource (7) per cluster, which contains all the information required to deploy the relevant modules for that cluster.
```

Comment:
You can write `to deploy them on the cluster whee they will run, as well as the asset details and the flow of data between the assets and the workload through the chosen modules.` (to specifically also write that the assets are declared separately in the plotter)

Also:
`The plotter controller generates a 'Blueprint' custom resource`  (the plotter is just a struct)